next.config.js

```diff
@@ -1,45 +1,36 @@
-const withImages = require('next-images');
 const path = require('path');
-const withTM = require('next-transpile-modules')([
-  '@formatjs/intl-relativetimeformat',
-  '@formatjs/intl-utils',
-  'react-intl',
-  'intl-format-cache',
-  'intl-messageformat-parser',
-  'intl-messageformat'
-]);
 
-const constructAlias = (config) => {
```

Comment:
Won't we need this for aliases to work ?

Reply:
ok yeah i will add both